src/VisualStudio/Core/Def/Progression/GraphQueries/SearchGraphQuery.cs

```diff
@@ -18,6 +19,8 @@ internal sealed partial class SearchGraphQuery(
 {
     public async Task<GraphBuilder> GetGraphAsync(Solution solution, IGraphContext context, CancellationToken cancellationToken)
     {
+        using var _ = Logger.LogBlock(FunctionId.GraphQuery_Search, KeyValueLogMessage.Create(LogType.UserAction), cancellationToken);
```

Comment:
Does this LogType.UserAction mean we'll actually get a telemetry event created?

Reply:
Cyrus asked something similar in the 17.14 PR. LogType doesn't seem to matter for telemetry, but this defaults the logLevel to LogLevel.Information. The telemetry logger ignores message with LogLevel < LogLevel.Information